test/output-tests/run_ml.expected

```diff
@@ -1,4 +1,14 @@
 Considering: import/gnucobol/tests/testsuite.src/run_ml.at:122:0
+run_ml.at-122-prog.cob:21.13:
+  18          01  0SpecialTAGName  PIC X(3) VALUE "abc".
+  19   
+  20          01  employee.
+  21 >            05 id            PIC 9(1)  value 1.
+----                ^
+  22              05 name          PIC X(10) value "Someone".
+  23              05 dept          PIC X(10) value "Marketing".
+>> Hint: Missing .
+
```

Comment:
Are those errors expected?

Reply:
Yes, the actual command in the autotest file passes `-fnot-reserved=id`. We currently don't parse these arguments.